core/src/avm2/globals/flash/display/NativeMenuItem.as

```diff
@@ -1,5 +1,7 @@
 package flash.display {
     import flash.events.EventDispatcher;
+
+    [API("661")]
```

Comment:
The docs disagree with `airglobal.swc` here, this may actually be `667` in reality.

Also, BTW, this class has a bunch of properties that are not added. (This is not a blocker, just mentioning, for completeness.)